### DISCUSSION TITLE
x-model with muliti-select in framework 7 Smart select

### DISCUSSION THREAD

**Discussion by wow-prime** [alpinejs/alpine#3861] [1. Help]
I have muliti-select in framework 7 Smart select like below picture
![image](https://github.com/alpinejs/alpine/assets/19904096/658b32e6-778b-4f66-9a27-a87b6fe14f41)

I select 3 options.
![image](https://github.com/alpinejs/alpine/assets/19904096/02ae8de2-7541-48bd-a989-44dfedc1fbf1)

But I only can get one value.
How can I get all value that I selected?
![image](https://github.com/alpinejs/alpine/assets/19904096/cd5c87f1-b863-4937-9617-bde4f8adaffc)

**Comment by ekwoka** [comment]
It's invalid HTML to have a `select` inside of an `a` https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#technical_summary

The simple answer though is that the select that that tool makes does not keep the actual select element in sync with it's state, so alpine can't actually see what the value is.


I'd probably get rid of jquery, and all that `var` as well. Major code smells that are likely making things harder than they should be.
